<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="launch_tab">
       <attribute name="title">
        <string>Launch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="launch_quit">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botta_launch">
            <property name="text">
             <string>BotTA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edain_launch">
            <property name="text">
             <string>Edain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotwk_launch">
            <property name="text">
             <string>RotWK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="launch_img">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="update_tab">
       <attribute name="title">
        <string>Update Configs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="cfg_quit">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="all_upd">
            <property name="text">
             <string>All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="botta_upd">
            <property name="text">
             <string>BotTA</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="edain_upd">
            <property name="text">
             <string>Edain</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rotwk_upd">
            <property name="text">
             <string>RotWK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="upd_img">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pref_tab">
       <attribute name="title">
        <string>Preferences</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="QScrollArea" name="pref_scroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>382</width>
             <height>581</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="7" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Mounting</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_17">
                 <item>
                  <widget class="QLabel" name="label_10">
                   <property name="text">
                    <string>Automatic</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="auto_mount">
                   <property name="toolTip">
                    <string>Enables automatic mounting of disc image</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_18">
                 <item>
                  <widget class="QLabel" name="label_13">
                   <property name="text">
                    <string>Mounting Executable</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="mount_exec">
                   <property name="toolTip">
                    <string>Path to mounting executable (e.g. DTCommandLine.exe, batchmnt.exe or similar)</string>
                   </property>
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Image</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="mount_image">
                   <property name="toolTip">
                    <string>Path to disc image to mount</string>
                   </property>
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_27">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_22">
                   <property name="text">
                    <string>Mounting Options</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="mount_opt">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Command line options for mounting with chosen executable. Leave empty if none are required.&lt;br&gt;(e.g. -m for DTCommandLine.exe, leave blank for batchmnt.exe)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_26">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_21">
                   <property name="text">
                    <string>Unmounting Options</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="umount_opt">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Command line options for unmounting with chosen executable. Leave empty if none are required. &lt;br&gt;(e.g. -u or -U for DTCommandLine.exe, /unmount or /unmountall for batchmnt.exe)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="layoutDirection">
                    <enum>Qt::RightToLeft</enum>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                   </property>
                   <property name="clearButtonEnabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>Specify Image When Unmounting</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="imspec_umount">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Whether the disc image should be specified when unmounting. Generally, this should be checked for any unmount command that does not unmount all mounted images.&lt;br&gt;(e.g. turn on if using -u with DTCommandLine.exe or /unmount with batchmnt.exe, leave off if using -U with DTCommandLine.exe or /unmountall with batchmnt.exe)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>BotTA</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>Installed</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QCheckBox" name="botta_installed">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables BotTA-related actions. Works only if the BotTA mod is installed and the correct path to its install directory is specified below&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>BotTA Path</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="botta_path_chooser">
                   <property name="toolTip">
                    <string>Path to BotTA directory (directory containing BotTa.lnk)</string>
                   </property>
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Edain</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Installed</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="edain_installed">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enables Edain-related actions.  Works only if the Edain mod is installed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Base Game</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Game Path</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="game_path_chooser">
                   <property name="toolTip">
                    <string>Path to game directory (directory containing lotrbfme2ep1.exe)</string>
                   </property>
                   <property name="text">
                    <string>Choose...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_9">
                   <property name="text">
                    <string>Default State</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="default_state">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;State in which the game is left after launch. Options are:&lt;br&gt;RotWK - patch 2.02 (Recommended for maximum compatibility)&lt;br&gt;Edain - patch 2.01 with Edain enabled&lt;br&gt;BotTA - patch 2.01&lt;br&gt; Last Launched - Whatever version was launched last&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_21">
                 <item>
                  <widget class="QLabel" name="label_15">
                   <property name="text">
                    <string>RotWK 2.02 version</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="rotwk_version">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Which version of 2.02 that should be applied when launching base game&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Swap game.dat File</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="dat_swap">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specifies whether the launcher should swap game.dat files. Turning this off means the 2.02 game.dat file will always be kept active, making it possible to play 2.01 and any mods requiring it without a disc.&lt;br&gt;(Recommended: On)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_20">
                 <item>
                  <widget class="QLabel" name="label_14">
                   <property name="text">
                    <string>Verify Active Configuration</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="verify_active">
                   <property name="toolTip">
                    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Whether the launcher should verify the integrity of the active version. Enabling this adds to the startup time.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Launcher</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_16">
                 <item>
                  <widget class="QLabel" name="label_11">
                   <property name="text">
                    <string>Terminate On Launch</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="kill_on_launch">
                   <property name="toolTip">
                    <string>Terminate launcher after starting the game</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_19">
                 <item>
                  <widget class="QLabel" name="label_12">
                   <property name="text">
                    <string>Show Console</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="show_console">
                   <property name="toolTip">
                    <string>Show console</string>
                   </property>
                   <property name="text">
                    <string/>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="pref_quit">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pref_save">
            <property name="maximumSize">
             <size>
              <width>75</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_23">
            <property name="minimumSize">
             <size>
              <width>225</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
